src/components/budget/budget-record.repository.ts

```diff
@@ -71,7 +71,7 @@ export class BudgetRecordRepository extends DtoRepository(BudgetRecord) {
     return !!result;
   }
 
-  async readOne(id: ID, session: Session, changeset?: ID) {
+  async readOneUnsecured(id: ID, session: Session, changeset?: ID) {
```

Comment:
I meant this change in the service files where there could be both a readOne & readOneUnsecured, both of which could be used depending on the need. readOne would become just shortcut for readOneUnsecured() + secure().

Repositories have no concept of securing objects so this suffix doesn't make sense here.